bin/docker/Dockerfile

```diff
@@ -3,14 +3,18 @@ FROM adoptopenjdk/openjdk11:alpine-jre
 ENV FC_LANG en-US LC_CTYPE en_US.UTF-8
 
 # dependencies
-RUN apk -U upgrade && apk add --update --no-cache bash ttf-dejavu fontconfig
+RUN apk -U upgrade &&  \
+    apk add --update --no-cache bash ttf-dejavu fontconfig curl java-cacerts && \
+    mkdir /app && mkdir /app/certs && \
```

Comment:
just a side note here, you can do `mkdir -p /app/certs` and it will create both `/app` and `/app/certs` if they don't exist

Reply:
great! didn't know that one!